rerun_py/rerun_sdk/rerun/log/lines.py

```diff
@@ -129,6 +132,222 @@ def log_line_strip(
         bindings.log_arrow_msg(entity_path, components=instanced, timeless=timeless, recording=recording)
 
 
+@log_decorator
+def log_line_strips_2d(
+    entity_path: str,
+    line_strips: Iterable[npt.ArrayLike] | None,
+    *,
+    identifiers: npt.ArrayLike | None = None,
+    stroke_widths: npt.ArrayLike | None = None,
+    colors: Color | Colors | None = None,
+    draw_order: float | None = None,
+    ext: dict[str, Any] | None = None,
+    timeless: bool = False,
+    recording: RecordingStream | None = None,
+) -> None:
+    r"""
+    Log a batch of line strips through 2D space.
+
+    Each line strip is a list of points connected by line segments. It can be used to draw
+    approximations of smooth curves.
+
+    The points will be connected in order, like so:
+    ```
+           2------3     5
+          /        \   /
+    0----1          \ /
+                     4
+    ```
+
+    Parameters
+    ----------
+    entity_path:
+        Path to the path in the space hierarchy
+    line_strips:
+        An iterable of Nx2 arrays of points along the path.
+        To log an empty line_strip use `np.zeros((0,0,3))` or `np.zeros((0,0,2))`
+    identifiers:
+        Unique numeric id that shows up when you hover or select the line.
+    stroke_widths:
+        Optional widths of the line.
+    colors:
+        Optional colors of the lines.
+        RGB or RGBA in sRGB gamma-space as either 0-1 floats or 0-255 integers, with separate alpha.
+    draw_order:
+        An optional floating point value that specifies the 2D drawing order.
+        Objects with higher values are drawn on top of those with lower values.
+        The default for lines is 20.0.
+    ext:
+        Optional dictionary of extension components. See [rerun.log_extension_components][]
+    timeless:
+        If true, the path will be timeless (default: False).
+    recording:
+        Specifies the [`rerun.RecordingStream`][] to use.
+        If left unspecified, defaults to the current active data recording, if there is one.
+        See also: [`rerun.init`][], [`rerun.set_global_data_recording`][].
+
+    """
+    recording = RecordingStream.to_native(recording)
+
+    colors = _normalize_colors(colors)
+    radii = _normalize_radii(stroke_widths)
+    radii = radii / 2.0
```

Comment:
this looks very wrong - are we doing something stupid in the rendering backend?

Reply:
It looks wrong because radii is stroke width. I'll rename the var above though.